edb/server/pgcon/pgcon.pyx

```diff
@@ -1590,6 +1605,18 @@ cdef class PGConnection:
                     self.connected = True
                     break
 
+                elif mtype == b'S':
+                    # ParameterStatus
+                    name, value = self.parse_parameter_status_message()
+                    if name == 'in_hot_standby':
+                        # Postgres 14 and above has this; lower Postgres is
```

Comment:
`ParameterStatus` will also be sent once the backend changes its state, so we need to handle it in the query loop also.